js/web/src/components/NavBar.js

```diff
@@ -197,11 +197,44 @@ const ConnectionDot = styled('span')(({ theme }) => ({
 }))
 
 const Logo = styled(NavLink)(() => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  padding: '7px 15px',
   position: 'absolute',
-  left: '50%',
-  top: '0',
-  transform: 'translateX(-50%)',
-  height: '60px',
+  width: '76px',
+  height: '43px',
+  left: 'calc(50% - 76px/2)',
```

Comment:
to center things w `position: absolute` we use the patter of `left: 50%, transformX(-50%)`. 

i think that should accomplish the same thing as the `76/2` and make the position consistent regardless of size